### PR TITLE
Slider - added tooltip

### PR DESCRIPTION
**Pull request by wtfil**

### CODE REVIEW

File: src/components/slider/slider.st.css [hunk 1]
```diff
@@ -223,16 +227,68 @@
...
+.tooltip.right {
+    left: value(tooltipOffset);
+    transform: translateY(-50%);
+    margin-top: 50%;
+}
+.tooltip.right:after {
+    right: 100%;
+    top: 50%;
+    transform: translateY(-50%);
+    border-right-color: value(tooltipColor);
```
Comment (tyv) [inline src/components/slider/slider.st.css]: I think there are a lot of options to make tooltips and arrows for them.
Can you make a quick research on that and then do the best proposal.

File: src/components/slider/slider.tsx [hunk 2]
```diff
@@ -122,7 +126,9 @@ export class Slider extends React.Component<SliderProps, SliderState> {
                 required={this.props.required!}
                 rtl={isRTL(this.context)}
                 step={this.props.step!}
-                tooltip={this.getTooltip()}
+                tooltip={this.state.relativeValue}
+                displayTooltip={this.props.displayTooltip!}
+                tooltipPosition={this.props.tooltipPosition || 'top'}
```
Comment (tyv) [inline src/components/slider/slider.tsx]: I think it should be in defaultProps

Comment (wtfil) [reply src/components/slider/slider.tsx]: true!

Comment (wtfil) [reply src/components/slider/slider.tsx]: fixed

**Pull request merged**